sdk/resourcemanager/azure-resourcemanager-authorization/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+- Removed `JsonFlatten` annotation in `RoleAssignmentCreateParameters`
```

Comment:
This is not needed, as it is codegen.

Instead you need to write to "Dependency Updates" part of api-version change.